src/worker/mod.rs

```diff
@@ -538,7 +546,7 @@ impl Worker {
         &self,
         run_id: &str,
         success: workflow_completion::Success,
-    ) -> Result<bool, CompleteWfError> {
+    ) -> Result<WFTReportOutcome, CompleteWfError> {
```

Comment:
Pedantic - Cannot tell with `Wf` vs `WFT` whether we should capitalize abbreviations or not

Reply:
Yeah I've been kinda inconsistent here. Clippy by default says don't, but I kinda like it. :man_shrugging: